src/components/ModalTreasury.vue

```diff
@@ -95,7 +95,7 @@ watch(open, () => {
         primary
         @click="handleSubmit"
       >
-        {{ $t('save') }}
+        {{ treasury.name ? $t('applyChanges') : $t('add') }}
```

Comment:
Shouldn’t this be `treasury?.name` , like the other checks ?

Reply:
Not sure where you mean. Treasury name should always be an empty string at minimum as far as I can see. 
<img width="568" alt="image" src="https://github.com/snapshot-labs/snapshot/assets/51686767/3ca21fd4-dab0-4dff-a5c5-33245392245f">

Reply:
here https://github.com/snapshot-labs/snapshot/blob/9b0eee90c14337250b09c1aadf2a02d433cf96c8/src/components/ModalTreasury.vue#L44C20-L44C20

it's checking for treasury existence

Reply:
Yeah that's not needed it this case. Will remove it